platform/vs/docker-ptf-sai.mk

```diff
@@ -3,7 +3,7 @@
 DOCKER_PTF_SAI = docker-ptf-sai.gz
 DOCKER_PTF_BASE = docker-ptf.gz
 $(DOCKER_PTF_SAI)_PATH = $(DOCKERS_PATH)/docker-ptf-sai
-$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_2) $(PYTHON3_THRIFT_2)
+$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_0_13_0) $(PYTHON3_THRIFT_0_13_0)
 $(DOCKER_PTF_SAI)_LOAD_DOCKERS += $(DOCKER_PTF_BASE)
 SONIC_DOCKER_IMAGES += $(DOCKER_PTF_SAI)
-SONIC_BUSTER_DOCKERS += $(DOCKER_PTF_SAI)
+SONIC_STRETCH_DOCKERS += $(DOCKER_PTF_SAI)
```

Comment:
It cannot be support in Buster? Prefer to use more modern OS version.

Reply:
the base docker docker_ptf in in stretch, so keep this one in stretch as well